README.md

```diff
@@ -12,6 +12,20 @@ This action handles the setup and teardown of an Azure Service Bus namespace for
           tag: PackageName
 ```
 
+The setup action also automatically propagates an environment variable called `EnvVarToCreateWithConnectionString_Restricted` with a dedicated connection string that only provides `Listen` and `Send` right. With that connection string, it is not possible to manage entities like queues, topics, subscriptions and rules. The connection string can be used to verify least-privilege scenarios.
+
 ## License
 
 The scripts and documentation in this project are released under the [MIT License](LICENSE).
+
+## Development
+
+Open the folder in Visual Studio Code. If you don't already have them, you will be prompted to install remote development extensions. After installing them, and re-opening the folder in a container, do the following:
+
+Run the npm installation
+
+```bash
+npm install
+```
```

Comment:
can't this be done as part of a [postCreate](https://containers.dev/implementors/json_reference/#lifecycle-scripts) callback?

Reply:
I think I tried it before and for some reason it didn't work, and I removed it again but I can't recall why.

Reply:
I think at the time I also added npm update which caused all sorts of troubles (or maybe I did it in the docker container. I added it and it works like a charm